editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -286,7 +286,13 @@ const RowForArrayControl = betterReactMemo(
     )
 
     const rowHeight = UtopiaTheme.layout.rowHeight.max
-    const { springs, bind } = useArraySuperControl(value, onSubmitValue, rowHeight, false)
+    const transformedValue = Array.isArray(value) ? value : []
```

Comment:
I think this should instead be:
```const transformedValue = Array.isArray(value) ? value : [value]```
As otherwise it runs the risk of all-too-easily accidentally binning off the original value if the control was incorrect.

Reply:
pushing a fix